src/open62541_compat.cpp

```diff
@@ -26,13 +26,10 @@
 #include <Utils.h>
 #include <boost/format.hpp>
 #include <boost/date_time.hpp>
-//#include <ASUtils.h>
 
-class alloc_error: public std::runtime_error
-{
-public:
-    alloc_error(): std::runtime_error("memory allocation exception") {}
-};
+#include <open62541_compat_common.h>
+
+
 				 
```

Comment:
Sorry, ignore this one - confused. Was the way github presented the source for viewing.